gnat2goto/gnat2goto.gpr

```diff
@@ -10,7 +10,8 @@ project GNAT2GOTO is
 
    for Main use ("gnat1drv.adb");
 
-   Common_Switches := ("-gnatg", "-g", "-gnateE");
+   --  Common_Switches := ("-gnatg", "-g", "-gnateE");
+   Common_Switches := ("-g", "-gnateE");
```

Comment:
Why?

Reply:
That should not have been committed. Whoops.